src/compressed-textures/ktx/loadKTX.ts

```diff
@@ -19,7 +19,7 @@ export const loadKTX = {
         priority: LoaderParserPriority.High,
     },
 
-    name: 'loadKTX2',
+    name: 'loadKTX',
```

Comment:
Other code still seems to be referencing `ktx2*`. Is the idea to just use the latest version for this identifier?

Reply:
will do a pass to make sure every thing in code named KTX rather than KTX2 as both are supported